ElectronicObserverTypes/Extensions/EquipmentTypesExtensions.cs

```diff
@@ -0,0 +1,198 @@
+
+		EquipmentTypes.CarrierBasedTorpedo => EquipmentTypeGroup.TorpedoBomber,
+		EquipmentTypes.JetTorpedo => EquipmentTypeGroup.TorpedoBomber,
+
+		EquipmentTypes.Interceptor => EquipmentTypeGroup.LandBasedFighters,
+
+		EquipmentTypes.LandBasedAttacker => EquipmentTypeGroup.LandBasedBombers,
+		EquipmentTypes.HeavyBomber => EquipmentTypeGroup.LandBasedBombers,
+
+		EquipmentTypes.CarrierBasedRecon => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.SeaplaneRecon => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.SeaplaneBomber => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.Autogyro => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.ASPatrol => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.FlyingBoat => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.SeaplaneFighter => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.LandBasedRecon => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.JetRecon => EquipmentTypeGroup.SeaplaneAndRecons,
+		EquipmentTypes.CarrierBasedRecon2 => EquipmentTypeGroup.SeaplaneAndRecons,
+
+		EquipmentTypes.RadarSmall => EquipmentTypeGroup.Radar,
+		EquipmentTypes.RadarLarge => EquipmentTypeGroup.Radar,
+		EquipmentTypes.RadarLarge2 => EquipmentTypeGroup.Radar,
+
+		EquipmentTypes.Sonar => EquipmentTypeGroup.ASW,
+		EquipmentTypes.DepthCharge => EquipmentTypeGroup.ASW,
+		EquipmentTypes.SonarLarge => EquipmentTypeGroup.ASW,
+
+		EquipmentTypes.ExtraArmor => EquipmentTypeGroup.Other,
+		EquipmentTypes.Engine => EquipmentTypeGroup.Other,
+		EquipmentTypes.AAShell => EquipmentTypeGroup.Other,
+		EquipmentTypes.APShell => EquipmentTypeGroup.Other,
+		EquipmentTypes.DamageControl => EquipmentTypeGroup.Other,
+		EquipmentTypes.ExtraArmorMedium => EquipmentTypeGroup.Other,
+		EquipmentTypes.ExtraArmorLarge => EquipmentTypeGroup.Other,
+		EquipmentTypes.Searchlight => EquipmentTypeGroup.Other,
+		EquipmentTypes.RepairFacility => EquipmentTypeGroup.Other,
+		EquipmentTypes.StarShell => EquipmentTypeGroup.Other,
+		EquipmentTypes.CommandFacility => EquipmentTypeGroup.Other,
+		EquipmentTypes.AviationPersonnel => EquipmentTypeGroup.Other,
+		EquipmentTypes.AADirector => EquipmentTypeGroup.Other,
+		EquipmentTypes.Rocket => EquipmentTypeGroup.Other,
+		EquipmentTypes.SurfaceShipPersonnel => EquipmentTypeGroup.Other,
+		EquipmentTypes.SearchlightLarge => EquipmentTypeGroup.Other,
+		EquipmentTypes.Ration => EquipmentTypeGroup.Other,
+		EquipmentTypes.Supplies => EquipmentTypeGroup.Other,
+		EquipmentTypes.TransportMaterial => EquipmentTypeGroup.Other,
+		EquipmentTypes.SubmarineEquipment => EquipmentTypeGroup.Other,
+
+		EquipmentTypes.LandingCraft => EquipmentTypeGroup.Transport,
+		EquipmentTypes.TransportContainer => EquipmentTypeGroup.Transport,
+		EquipmentTypes.SpecialAmphibiousTank => EquipmentTypeGroup.Transport,
+
+		EquipmentTypes.VTFuse => throw new NotImplementedException(),
+		_ => throw new NotImplementedException(),
```

Comment:
Set these 2 as EquipmentTypeGroup.Unknown, otherwise EO is gonna blow up if they ever add new equip types.